client/js/components/map/map/DpOlMap.vue

```diff
@@ -490,6 +491,11 @@ export default {
     this.olMapState.drawStyles = this.getDrawStyles()
 
     this.registerFullscreenChangeHandler()
+
+    // Selectors/Hooks needed for Cypress Testing.
+    document.querySelector('.ol-full-screen-false')?.setAttribute('data-cy', 'map:toggleFullScreen')
+    document.querySelector('.ol-zoom-in')?.setAttribute('data-cy', 'map:zoomIn')
+    document.querySelector('.ol-zoom-out')?.setAttribute('data-cy', 'map:zoomOut')
```

Comment:
even though, the hooks are requested. I think we should not do that. With that  we start to test libs wich are not maintained by us.

Reply:
https://github.com/demos-europe/demosplan-core/pull/3099/commits/d1f839ace7a4926f2fc571a1ba030437f6d6acfd